system-variables.md

```diff
@@ -996,6 +996,20 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
 - This variable is used to set whether the optimizer executes the optimization operation of pushing down the aggregate function to the position before Join, Projection, and UnionAll.
 - When the aggregate operation is slow in query, you can set the variable value to ON.
 
+### `tidb_opt_limit_push_down_threshold`
+
+- Scope: SESSION | GLOBAL
+- Default value: `100`
+- Range: `[0, 2147483647]`
+- This variable is used to push down the Limit and TopN operators to the threshold in TiKV.
```

Comment:
`This variation is used to set the threshold that determines whether push the Limit or TopN operators down to TiKV`

Reply:
Rest LGTM

Reply:
```suggestion
- This variable is used to set the threshold that determines whether to push the Limit or TopN operator down to TiKV.
```